meetings/2019-03-f2f.md

```diff
@@ -0,0 +1,247 @@
+  * Members in good standing will be asked to rejoin.
+  * Members are under no obligation to rejoin
+  * Year starts on Jan, 1st.
+  * Members can resign at any time
+  * We need a concept of "Good Standing" (see for example [W3C's definition](https://www.w3.org/2005/10/Process-20051014/process.html#good-standing)), still TBD.
+* **AI(@tobie): open a pull request against the AC's work mode document to reflect agreed upon term durations.**
+* Current AC members approve any new AC member
+  * Consensus based process
+  * AC want to ensure a diverse group of representative (diversity of location, thought, industry, gender, disability, race, etc)
+  * "It is the AC's duty to ensure the AC is as representative and diverse as possible. When AC membership places are available, AC members commit to proactively reaching out to groups who are not currently represented. AC members will use Single Transferable Vote to select new members in a secret ballot."
+  * **AI(@tobie): open a pull request against the AC's workmode document to include a description of how new AC members are elected.**
+
+### Is AMP a product or an Open Source Project
+
+* Currently the perception is that it is a Google Product.
+* Only Google employees can merge requests for now (plan is for this to move to <33%)
+* Majority or changes are by Google employees
+* AMP Cache is not open source. (Reason: it's too tied to infrastructure to make sense open sourcing it. It couldn't be re-used.)
+* The perception is that AMP is beneficial to Google - not for the open web.
+* AMP needs to be separated - and seen to be separated - from Google. Too much interplay between Search, Browser, and AMP.
+* AC may advise both the TSC (open source) and the wider community (products).
+* AMP must share their user research openly.
+* User research data must be available to the AC/TSC to allow them to make decisions.
+* **AI(@edent, @ljwatson): draft a paper on results of user research.**
+
+### AMP Letter
+
+Discuss AMP Letter in preparation for meeting with signatories.
+* disclosure: 2 members of the AC present are AMP letter signatories.
+* URL Problem
+  * Dodgy sites get to use the google.com URL
+     * Users find this confusing
+     * It is seen as breaking the web
+     * This is an unfortunate byproduct of early design decisions
+  * WebPackaging being the answer?
+     * Not a standard (yet)
+     * Not supported in all browsers (Firefox, Safari)
+  * Given that the URL problem is a user-unfriendly pattern, and that two major browsers are unlikely to support WebPackaging, the AC needs to provide advice to the TSC on fixing this.
+* Issue with pref search treatment
+  * How do we account for web pages "cheating" on performance test? Dieselgate style.
+  * You have to use AMP to be in Google News Carousel. Discovery experience. This is perceived as discriminatory.
+  * Developers don't want multiple pipelines for their content. They have been incentivised by being at the top of search results.
```

Comment:
Developers don't want multiple pipelines for their content. They have been incentivized by being at the top of search results.